runelite-client/src/main/java/net/runelite/client/plugins/banktags/TagManager.java

```diff
@@ -79,6 +80,31 @@ String getTagString(int itemId, boolean variation)
 		return config;
 	}
 
+	public int getQuantityOfItemForTag(String tag, int itemId, boolean variation)
+	{
+		itemId = getItemId(itemId, variation);
+
+		String config = configManager.getConfiguration(CONFIG_GROUP, ITEM_QUANTITY_KEY_PREFIX + itemId + "." + tag);
```

Comment:
I don't think this should display of a config isn't found

Reply:
In the overlay class, it doesn't display quantities of 1, so if there is no config found, it will not display a quantity.

Reply:
My original statement holds true, we don't really need to try and cast it to a number. If you convert it to the stacksize before hand and store it like that, it doesn't matter.